data/scene/lodglobes/earth/earth.mod

```diff
@@ -1,4 +1,5 @@
-earthEllipsoid = {6378137.0, 6378137.0, 6356752.314245} -- Earth's radii
+--earthEllipsoid = {6378137.0, 6378137.0, 6356752.314245} -- Earth's radii
+earthEllipsoid = {6378137.0, 6378137.0, 6378137.0} -- Earth's radii
```

Comment:
Right now to avoid the ellipsoid/ray intersection. It will moved back to the ellipsoid radii before the release.